acceptance/support/data-registry.ts

```diff
@@ -0,0 +1,52 @@
+import {AxiosResponse} from "axios";
+import {Attribute} from "./test-constants";
+
+/**
+ * Component to store intermediate data during Cucumber test execution.
+ */
+export default class DataRegistry {
```

Comment:
This is a smarter Map and I like it!

Reply:
Actually I wanted to use an async local storage, but I won't parallelize the test execution, so it doesn't really matter at this point. (Btw, thanks! :) )